src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -111,13 +111,13 @@ private GFF3Feature createLandmarkFeature(String name, Entry entry) {
         Map.of("ID", name, "Is_circular", "true"));
   }
 
-  private GFF3Feature transformFeature(String accession, Feature ffFeature, Optional<String> gene) {
+  private List<GFF3Feature> transformFeature(String accession, Feature ffFeature, Optional<String> gene) {
     Map<String, String> qualifierMap = ConversionUtils.getFF2GFF3QualifierMap();
 
     String source = ".";
     String score = ".";
 
-    Map<String, String> attributes =
+    Map<String, String> base_attributes =
```

Comment:
Suggestion to use camel case for variables across our application **baseAttributes**